python/dex/__init__.py

```diff
@@ -32,8 +34,7 @@ def _from_ptr(cls, ptr):
     return self
 
   def __del__(self):
-    if api is None or api.nofree: return
-    api.destroyContext(self)
+    pass  # TODO: free
```

Comment:
Why can't we call `destroyContext` anymore?

Reply:
also a mistake